Casks/v/vezer.rb

```diff
@@ -0,0 +1,32 @@
+cask "vezer" do
+  version "1.9.6,776,2"
+  sha256 "8e2f656f527d1ff410d0f7f594c3922e14105b5eda3f184e8e5e2f02f175b431"
+
+  url "https://dl.imimot.com/Vezer#{version.csv.first}_#{version.csv.second}v#{version.csv.third}.zip"
+  name "Vezér"
+  desc "Control and synchronisation of MIDI, OSC or DMX"
+  homepage "https://imimot.com/vezer/"
+
+  livecheck do
+    url "https://imimot.com/versions/VezerAppcast.php?beta=0&app_version=0"
+    regex(/Vezer(?:\d+(?:\.\d+)+)[._-](?:\d+)v(\d+)\.zip/i)
+    strategy :sparkle do |item|
+      version_suffix = item.url.match(regex).captures.first
+      next if version_suffix.blank?
+
+      "#{item.short_version},#{item.version},#{version_suffix}"
+    end
```

Comment:
```suggestion
    regex(/Vezer(?:\d+(?:\.\d+)+)[._-](\d+(?:v\d+)?)\.zip/i)
    strategy :sparkle do |item|
      match = item.url.match(regex)
      next if match.blank?

      "#{item.short_version},#{match[1]}"
    end
```

It might make sense to capture the suffix as an optional part of the build number since it may not always be present.